protocol.md

```diff
@@ -229,6 +229,17 @@ interface Location {
 }
 ```
 
+#### Preview Location
+
+Represents a location inside a resource, such as a line inside a text file.
+```typescript
+interface PreviewLocation extends Location {
+	uri: string;
+	range: Range;
+	linePreview: string;
```

Comment:
They use the term `contents` for marked string, maybe that'd be good here instead of `linePreview`.